image-library/image_scraper/fark_scraper.py

```diff
@@ -47,4 +47,119 @@ def getdata(url):
 		print(a)
 
 
+#%%
```

Comment:
All this selenium will likely inform how we can get around 403 errors when crawling fark.